AES.Entities/Tables/Availability.cs

```diff
@@ -1,41 +1,162 @@
 ﻿using System;
 using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
 
 namespace AES.Entities.Tables
 {
     public class Availability
     {
+
+        public Availability()
+        {
+
+            DTSundayStart =
```

Comment:
I know Re#er did it this way, but this is a lot of lines for such a simple statement. Better to just chain them side-by-side, and maybe break down to two lines.